paimon-core/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -568,6 +568,27 @@ public class CoreOptions implements Serializable {
                             "Only used to force TableScan to construct 'ContinuousCompactorStartingScanner' and "
                                     + "'ContinuousCompactorFollowUpScanner' for dedicated streaming compaction job.");
 
+    public static final ConfigOption<StreamReadType> STREAMING_READ_FROM =
+            key("streaming-read-from")
```

Comment:
Could we use `streaming-read-mode` and the value of this option could be `FILE` and `LOG`?

Reply:
have been modified